pkg/offline/offline.go

```diff
@@ -157,6 +157,146 @@ func WithQueue(filepath string, syncLimit int) (heartbeat.HandleOption, error) {
 	}, nil
 }
 
+// Sync returns a function to send queued heartbeats to the WakaTime API.
+func Sync(filepath string, syncLimit int) func(next heartbeat.Handle) error {
```

Comment:
`syncLimit` should just be `limit` because this code has nothing to do with syncing.. it's just popping multiple (up to 24) heartbeats from the offline db.